cypress/support/step_definitions/dictionary/addConceptInDictionaryPage/addconceptInDictioanryPage.ts

```diff
@@ -0,0 +1,69 @@
+/// <reference types="cypress" />
+/// <reference types="../../../" />
+import {
+  Given,
+  Then,
+  When
+} from "cypress-cucumber-preprocessor/steps";
+import { getDictionaryId, getUser, getConceptId } from "../../../utils";
+
+
+Given("the user is on the view dictionary concepts page", () => {
+  cy.visit(`/users/${getUser()}/collections/${getDictionaryId()}/concepts/`);
+});
+
+When('the user clicks the "Add concepts" button', () => {
+  cy.findByTitle("Add concepts").click();
+});
+
+Then('the user should be on the "Import existing concept" page', () => {
+  cy.url().should("contain", "/orgs/CIEL/sources/CIEL/concepts/");
+});
+
+Given('the user is on the "Import existing concept" page', () => {
+  cy.url().should("contain", "/orgs/CIEL/sources/CIEL/concepts/");
```

Comment:
I guess this one should be
```suggestion
  cy.visit("/orgs/CIEL/sources/CIEL/concepts/");
```